src/main/java/com/CezaryZal/api/body/BodySizeController.java

```diff
@@ -13,50 +16,45 @@
 @RequestMapping("/api/body")
 public class BodySizeController {
 
-    private BodySizeService bodySizeS;
+    private BodySizeService bodySizeService;
 
-    public BodySizeController(BodySizeService bodySizeS) {
-        this.bodySizeS = bodySizeS;
+    public BodySizeController(BodySizeService bodySizeService) {
+        this.bodySizeService = bodySizeService;
     }
 
-    @ApiOperation(value = "This will get a `BodySize by id", notes = "In this method you will receive a body measurement by id")
+    @ApiOperation(value = "This will get a `BodySize` by id", notes = "In this method you will receive a body measurement by id")
     @GetMapping("/{id}")
-    public BodySize getBodyById (@PathVariable Long id){
-        return bodySizeS.getBodyById(id);
+    public BodySizeDto getBodySizeDtoById(@PathVariable Long id) {
+        return bodySizeService.getBodySizeDtoById(id);
     }
 
     @ApiOperation(value = "This will get a last date measure by user id")
     @GetMapping("/date/user-id/{userId}")
-    public LocalDate getDateLastMeasureByUserId(@PathVariable Long userId){
-        return bodySizeS.getDateLastMeasureByUserId(userId);
+    public LocalDate getDateLastMeasureByUserId(@PathVariable Long userId) {
+        return bodySizeService.getDateLastMeasureByUserId(userId);
     }
 
     @ApiOperation(value = "This will be get a list dates by user id")
     @GetMapping("/dates/user-id/{userId}")
-    public List<LocalDate> getListDatesByUserId (@PathVariable Long userId){
-        return bodySizeS.getListDatesByUserId(userId);
+    public List<LocalDate> getListDatesByUserId(@PathVariable Long userId) {
+        return bodySizeService.getListDatesByUserId(userId);
     }
 
     @ApiOperation(value = "This will be get a `BodySize` by date and user id")
     @GetMapping("/{date}/{userId}")
-    public BodySize getBodyByDateAndUserId (@PathVariable String date, @PathVariable Long userId){
-        return bodySizeS.getBodyByDateAndUserId(date, userId);
-    }
-
-    @ApiOperation(value = "This will get a list `BodySize`, all records")
-    @GetMapping
-    public List<BodySize> getAll(){
-        return bodySizeS.getAll();
+    public BodySizeDto getBodyByDateAndUserId(@PathVariable String date, @PathVariable Long userId) {
+        return bodySizeService.getBodyDtoByDateAndUserId(date, userId);
     }
 
     @PostMapping
-    public boolean addBody (@RequestBody BodySize bodySize){
-        return bodySizeS.addBody(bodySize);
+    public String addBody(@RequestBody BodySizeDao bodySize) {
```

Comment:
To co powinno przychodzić do endpointu to bardziej BodySizeDTO i zwrócenie samego stringa też jest takie sobie. 
Ja bym tu ustawił żeby poszedł dodatkowo określony status na 204.

Reply:
Albo string albo boolen, tylko jako potwierdzenie że jest ok.
A status też dodam przez RequestEntity ale jak widać narazie tego nie wrzucałem

Reply:
204 | No content | Brak zawartości – serwer zrealizował zapytanie klienta i nie potrzebuje zwracać żadnej treści
-- | -- | --